package/doc/sphinx/source/documentation_pages/loading_files/amber.rst

```diff
@@ -0,0 +1,37 @@
+.. _loading_amber:
+
+###################################
+Loading Amber files with MDAnalysis
+###################################
+
+MDAnalysis can read etc etc etc from Amber
+
+
+.. _load_amber_top:
+
+Loading Amber TOP
+-----------------
+
+TOP files etc etc etc
+
+.. _load_amber_trj:
+
+Loading TRJ files
```

Comment:
I would talk about NCDF first – that's much better supported. See current docs (which I recently rewrote a little bit)